src/components/AccountSelector.tsx

```diff
@@ -0,0 +1,90 @@
+import type { KeyringPair } from '@polkadot/keyring/types'
+import PropTypes from 'prop-types'
+import React, { useState, useEffect } from 'react'
+import { Dropdown, DropdownButton } from 'react-bootstrap'
+import { useSubstrate } from '../substrate'
+
+type Props = {
+  setAccountAddress: (address: string) => void
+}
+
+function Main({ setAccountAddress }: Props) {
+  const { keyring } = useSubstrate()
+  const [accountSelected, setAccountSelected] = useState('')
+
+  const keyringOptions = keyring?.getPairs().map((account: KeyringPair) => {
+    return {
+      key: account?.address,
+    }
+  })
+
+  const initialAddress = keyringOptions?.length > 0 ? keyringOptions[0].key : ''
+
+  useEffect(() => {
+    setAccountAddress(initialAddress)
+    setAccountSelected(initialAddress)
+  }, [setAccountAddress, initialAddress])
+
+  const onChange = (account: HTMLInputElement) => {
+    setAccountAddress(account.innerText)
+    setAccountSelected(account.innerText)
+  }
+
+  const Title = () => {
+    return (
+      <label
+        style={{
+          fontSize: '12px',
```

Comment:
We haven't decided on a css strategy yet. Should we use themed components, inline css, style tags, or another option?

Reply:
Yes... I would suggest to use Styled Components.